tools/bootstrap-network.sh

```diff
@@ -53,6 +53,11 @@ export CHUTNEY_NETWORK="$CHUTNEY_PATH/networks/$NETWORK_FLAVOUR"
 
 "$CHUTNEY" stop "$CHUTNEY_NETWORK"
 
+if ! "$CHUTNEY" supported "$CHUTNEY_NETWORK"; then
```

Comment:
Should this code go before the "stop" command?
Otherwise, the "stop" command might fail.

Reply:
I'm thinking about the case where we switch Tor versions between one run and the next.  I'm guessing that in this case, we'd be better off _trying_ to stop the network, even if it fails.

Reply:
Fair enough.